content/influxdb/v2.3/process-data/get-started.md

```diff
@@ -113,6 +141,46 @@ from(bucket: "example-bucket")
 
 _See [Common tasks](/influxdb/v2.3/process-data/common-tasks) for examples of tasks commonly used with InfluxDB._
 
+{{% cloud-only %}}
+
+### Process data with invokable scripts
```

Comment:
This sections feels like it should be in the "Create a task to run an invokable scripts" doc rather than in the "Get started with InfluxDB tasks" doc.

Reply:
> This sections feels like it should be in the "Create a task to run an invokable scripts" doc rather than in the "Get started with InfluxDB tasks" doc.

Thanks, I debated that. As it turns out, this might be the only kind of task going forward. In any case, this PR was premature and I'm waiting to hear more.